gems/pending/metadata/ScanProfile/modules/HostScanItemNteventlog.rb

```diff
@@ -72,7 +72,7 @@ def hostd_log_to_hashes(log, filter = nil)
         :level     => level,
         :source    => source,
         :message   => message,
-        :uid       => Digest::MD5.hexdigest("#{generated} #{name} #{level} #{source} #{message}")
+        :uid       => Rails.application.config.digest_class.hexdigest("#{generated} #{name} #{level} #{source} #{message}")
```

Comment:
If the digest changes it will cause some churn in the next inventory refresh because of the `:uid` field change ([app/models/ems_refresh/save_inventory.rb#L312](https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_refresh/save_inventory.rb#L312)) but otherwise this is ok.  👍